### PR TITLE
Fixes #81 : Porting AD7718

### LINKED ISSUE #81: Port AD7718
Have to implement AD7718 in the Android version!

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Ported AD7718 sensor from the python version!

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ dependencies {
     compile 'com.github.bumptech.glide:glide:3.7.0'
     compile 'de.hdodenhof:circleimageview:2.1.0'
     compile 'org.apache.commons:commons-math3:3.6.1'
+    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.5'
```
Comment (viveksb007) [inline app/build.gradle]: can't it be like compile 'org.apache.commons:commons-lang3:3.5'  ??
Also I see common math version to be 3.6.1, use that if its available.

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+            if ((stat & (1 << a)) == 1) {
+                sb.append(P[a]);
+            } else {
+                sb.append(N[a]);
+            }
+        }
+        Log.d(TAG, stat + ", " + sb.toString());
+    }
+
+    public float convert_unipolar(float x) {
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: Avoid using '_' in method or Variables name in java. should be "convertUnipolar" instead.

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Padmal on 5/3/17.
+ */
+
+public class AD7718 {
+
+    double VREF = 3.3;
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: make all variables used private which are specific to this class.

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+    }
+
+    public boolean readRawVoltage(String channel) throws IOException, InterruptedException {
+        if (startRead(channel)) {
+            return false;
+        }
+        Thread.sleep(150);
+        return fetchRawData(channel);
+    }
+
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: And give a meaning full name wherever you can Please. "bv" doesn't give a clue to what it does.

File: app/build.gradle (hunk 1 above)
Comment (CloudyPadmal) [inline app/build.gradle]: Fixed!! :)

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java (hunk 3 above)
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: Fixed!! :)

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java (hunk 2 above)
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: Fixed!! :)

File: app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java (hunk 4 above)
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/AD7718.java]: Fixed!! :)

**Comment by viveksb007** [comment]
@CloudyPadmal Please go through http://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/ to follow best practices.
And add reference to issue in PR description.

**Pull request merged**